README.md

```diff
@@ -1,3 +1,6 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+# Instructions
+
+To clone this project and run it locally: 
+You will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=, with the correct database name for that environment.
```

Comment:
You need to be explicit in telling the user, what each the values of PGDATABASE should be for each context.